### ISSUE TITLE
Aegis will not open its own .json backup file, claims password is wrong.

### ISSUE THREAD

**Issue by cyan-lime** [beemdevelopment/Aegis#308]
I'd be interested to see if people can reproduce this. Today I migrated between two Android roms. I backed up my Aegis database as a .json using the app thinking I would be able to import it on the new rom. To my horror, however, I got a message stating 'Incorrect password. Make sure you didn't mistype your password.' I did not mistype my password. I'm now locked out of several important accounts of mine so I have a great vested interest in seeing this problem solved and hopefully recovering my 2FA codes.

**Comment by chiraag-nataraj** [comment]
Hmm, I just tried importing an (encrypted) file I exported yesterday and everything worked fine (password worked and I was asked which entries I would like to import, at which point I backed out of the process).

Which version are you using?

**Comment by geoffroy-noel-ddh** [comment]
I have a similar issue. I cannot import the aegis_export.json file I've exported earlier today. Settings > Import from file > Aegis > select the file_export from sd card and I shows the following message:

"Error: file not found"

**Comment by alexbakker** [comment]
Please fill out [the template](https://github.com/beemdevelopment/Aegis/blob/master/.github/ISSUE_TEMPLATE/bug.md) when creating bug reports. It's there so that we don't have to continuously ask users for basic information about their device and Aegis installation.

Can you open your ``aegis_export.json`` file and compare the structure to [testdata/aegis_export.json](https://github.com/beemdevelopment/Aegis/blob/master/testdata/aegis_export.json)? Do notice any differences? Also, can you still reliably reproduce this issue by creating a new vault, exporting it and then trying to import it again?

@geoffroy-noel-ddh That sounds unrelated. Can you create a new issue for that? What happens if you move that file to a different location and import it from there?

**Comment by seularts** [comment]
I have the exact same issue as geoffroy-noel-ddh: "Error: file not found"
I lost my previous device and I am trying to import it into a new one. O checked the structure of the json mentioned above and I have one extra line of code: "repaired": true

**Comment by geoffroy-noel-ddh** [comment]
@alexbakker thanks for asking. Don't think it's worth creating a new ticket as I can't easily reproduce it.

Summary: I had the latest version of Aegis on android 4 days ago. I exported as a json file to Google Drive. After a factory reset of my phone (and re-installing Aegis, obviously), I got the 'Error: file not found' each time I tried to import that same file from the external sdcard. If I export again into a second file, directly to the sdcard this time, alongside the first one then I can re-import it. In the file browser the two files had different icons though, although the extension was json. It might be a filesystem permission... I'm sure the file content was valid.

Anyway, if anyone else experience this problem, I managed to solve this by re-import the file via GDrive.
